src/components/app/ListOfPublishedProfiles.css

```diff
@@ -0,0 +1,82 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+.publishedProfiles {
+  /* Box */
+  width: 95%;
+  box-sizing: border-box;
+  padding: 3em 1em;
+  border: 1px solid #ccc;
+  margin: auto;
+
+  /* Other */
+  background: #fff;
+  border-radius: 3px;
+  box-shadow: 0 5px 25px #0b1f50;
+  font-size: 13px;
+  line-height: 1.5;
+}
+
+.publishedProfilesList {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.publishedProfilesDate {
+  position: absolute;
+}
+
+.publishedProfilesName {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+
+.publishedProfilesName,
+.publishedProfilesList .profileMetainfoSummary {
+  padding-left: 10em;
+}
+
+.publishedProfilesListItem {
+  padding: 5px 0;
+}
+
+.publishedProfilesLink {
+  display: block;
+  padding: 2px 5px;
+  color: inherit;
+  text-decoration: none;
+}
+
+.publishedProfilesLink:hover {
+  background: var(--grey-90-a10);
+}
+
+/* In larger screens, we remove the flex characteristics but force some width
+ * for the first element.
+ * We also add some more padding and change how the width is computed.
+ *
+ * 47em is 752px (there's a 16 multiplicator), 29em is 464px, which were the
+ * right values when testing on Linux. Fonts are usually larger on Linux for
+ * the profiler, so in other OSes we could decide that the value is too large,
+ * but this is fine. */
+@media (min-width: 47rem) {
```

Comment:
Can you explain the choice of using `rem` here rather than pixels? It seems like a lot of explanations of sizing rather than just using pixel values.

Reply:
I'll have to double check. I read blog posts specifically about this but this was some weeks ago now :)
I think this is for reasons that may be unrelevant for our project because we use px everywhere already.

Reply:
I changed this to px units.